addons/mail/static/src/components/message/message.xml

```diff
@@ -53,20 +52,22 @@
                         </t>
                         <t t-else="">
                             <t t-if="messageView.isActive and messageView.message.date">
-                                <small class="o_Message_date o_Message_sidebarItem mt-1 mx-1" t-att-class="{ 'o-message-selected text-600': messageView.isSelected, 'text-500': !messageView.isSelected }">
+                                <small class="o_Message_date o_Message_sidebarItem mt-3 mx-1 text-muted opacity-50">
                                     <t t-esc="messageView.message.shortTime"/>
                                 </small>
                             </t>
                             <t t-if="!messageView.isActive and messageView.messageSeenIndicatorView">
-                                <MessageSeenIndicator className="'o_Message_seenIndicator me-1'" record="messageView.messageSeenIndicatorView"/>
+                                <div t-att-class="{ 'position-absolute bottom-0 mb-1': !messageView.inChatter }">
+                                    <MessageSeenIndicator className="'o_Message_seenIndicator me-1'" record="messageView.messageSeenIndicatorView"/>
+                                </div>
                             </t>
                         </t>
                     </div>
-                    <div class="o_Message_core flex-grow-1 me-4">
+                    <div class="o_Message_core flex-grow-1">
                         <t t-if="!messageView.isSquashed">
-                            <div class="o_Message_header d-flex flex-wrap align-items-baseline ms-2">
+                            <div class="o_Message_header d-flex flex-wrap align-items-baseline" t-att-class="{ 'justify-content-end': messageView.inChatWindow and messageView.message.isCurrentUserOrGuestAuthor }">
                                 <t t-if="messageView.message.author">
-                                    <strong class="o_Message_authorName o_redirect me-2 text-truncate" t-att-class="{'o_cursor_pointer': messageView.hasAuthorOpenChat}" t-att-role="messageView.hasAuthorOpenChat ? 'button' : ''" t-att-tabindex="messageView.hasAuthorOpenChat ? '0' : ''" t-on-click="messageView.onClickAuthorName" t-att-title="messageView.authorTitleText">
+                                    <strong class="o_Message_authorName o_redirect me-1 text-truncate" t-att-class="{ 'o_cursor_pointer': messageView.hasAuthorOpenChat, 'd-none': messageView.inChatWindow and (messageView.message.isCurrentUserOrGuestAuthor or (messageView.messageListViewMessageViewItemOwner and  messageView.messageListViewMessageViewItemOwner.messageListViewOwner.thread.channel_type === 'chat')) }" t-att-role="messageView.hasAuthorOpenChat ? 'button' : ''" t-att-tabindex="messageView.hasAuthorOpenChat ? '0' : ''" t-on-click="messageView.onClickAuthorName" t-att-title="messageView.authorTitleText">
```

Comment:
Replace `d-none` by `t-if`

Reply:
Also the condition is long and hard to understand at a glance.
Easily fixed by introducing the proper concept, e.g. with new (computed) field `MessageView/isShowingAuthorName`